tripal_chado/src/Plugin/TripalStorage/ChadoStorage.php

```diff
@@ -1476,7 +1512,7 @@ static public function drupalEntityIdLookupCallback($context) {
     $prop_storage_settings = $context['prop_type']->getStorageSettings();
     $fkey = $prop_storage_settings['fkey'] ?? NULL;
     if (!$fkey) {
-      // Maybe throw an exception here so developers know they forgot the 'fkey'
+      // Maybe throw an exception so developers know they forgot the 'fkey'.
```

Comment:
```suggestion
      // @todo Throw an exception so developers know they forgot the 'fkey'.
```
This is originally my comment, maybe I will fix someday 🙈